cli/run_server.py

```diff
@@ -8,6 +8,20 @@
 use_hivemind_log_handler("in_root_logger")
 logger = get_logger(__file__)
 
+import re
+
+
+def parse_size_as_bytes(size: str) -> int:
+    """parse human-readable data size e.g. 1.5GB, based on https://stackoverflow.com/a/42865957/2002471"""
```

Comment:
This cuts some corners (e.g. 1GB = 1GiB). We can get it right all the time by using https://pypi.org/project/humanfriendly/ , but I'm not sure it justifies the extra depenedcy

@borzunov any preferences?

Reply:
* I'm okay with adding this light dependency.
* I'm not okay (weakly) with the same behavior for `GB` and `GiB`.

To support both `GB` and `GiB` in a correct way, you can actually simplify the code to do smth like:

```python
units = ['B', 'KB', 'MB', 'GB', 'TB', 'PB']
unit = unit.upper()
if 'I' in unit:
    unit = 1024 ** units.index(unit.replace('I', ''))
else:
    unit = 1000 ** units.index(unit)
```

In other words, you can use the same array of units, just with different bases :)